static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -901,3 +903,80 @@ $caption-button-size: 32px;
         }
     }
 }
+
+/****************
+ * FOOTER
+ ****************/
+
+.l-footer {
+    // these two lines are needed so that the footer sits over the nav
+    position: absolute;
+    z-index: 9999;
+    width: 100%;
+}
+.footer__back-to-top__container {
+    @include content-gutter();
+    background-color: $guardian-brand;
+
+    a:hover > &,
+    a:focus > & {
+        background-color: $guardian-brand-dark;
+    }
```

Comment:
Discussed in person: Leaving in this styling in preparation for the desktop version (coming soon)